app/forms/declaration/destinationCountries/DestinationCountries.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms.declaration.destinationCountries
+
+import play.api.data.Form
+
+object DestinationCountries {
+  val formId = "DestinationCountries"
+
+  val limit = 99
+
+  def supplementaryForm(): Form[DestinationCountriesSupplementary] = Form(DestinationCountriesSupplementary.mapping)
```

Comment:
I think that these methods should be placed in proper companion objects as well. The usage is not intuitive.

Reply:
As above